int.ts

```diff
@@ -99,7 +99,9 @@ module J2ME {
    *             +--------------------------------+
    *             | Caller FP                      |
    *             +--------------------------------+
-   *             | Callee Method Info | Marker    |
+   *             | Frame Type                     |
```

Comment:
If we're storing this in a separate slot now we should update the FrameType constants to just use low numbers. (produces faster switch statements when the numbers are dense)